### ISSUE TITLE
medaka variant outputs p-value <= 0.0

### ISSUE THREAD

**Issue by menickname** [nanoporetech/medaka#149]
I am currently running a reference-based bacterial genome assembly pipeline including medaka consensus, medaka variant, followed by longshot [https://github.com/pjedge/longshot](url). However, an error occurs during this longshot step resulting from a p-value <= 0.0 in the previous steps. 

`thread 'main' panicked at 'assertion failed: p <= 0.0'`

I have been looking on the medaka repository, but no clear solution was found. Is there any additional option to be included to avoid medaka variant outputting variants with p-value <= 0.0?

Thank you in advance.
Best regards,
Nick Vereecke

**Comment by cjw85** [comment]
Hi @menickname,

Is there a particular reason you are running longshot after medaka? I've only briefly studied the the longshot code so can only really guess at what is happening.

If you are giving medaka's vcf to longshot for it to run it's genotyping (and phasing) calculations I don't believe longshot reads anything from the vcf other than the identity of the variants. As such the error reported above looks to be more of a longshot issue than a medaka one.

**Comment by menickname** [comment]
Dear @cjw85 

Thank you for your reply. I have been using longshot because of its use in current artic network SARS-CoV-2 workflow. I tried to use a similar workflow in an other analysis pipeline, however not resulting in success. 

I have also posed the question on the longshot repository on github and still waiting for reply on that. I also think the issue is rather with longshot and not with medaka.
